x-pack/plugins/security/public/authentication/logout/logout_app.ts

```diff
@@ -8,6 +8,8 @@
 import { i18n } from '@kbn/i18n';
 import type { CoreSetup, HttpSetup } from 'src/core/public';
 
+import { SecurityTelemetryService } from '../../telemetry';
```

Comment:
Nit: to stay consistent with platform guidelines, we want to use the term "usage collection" instead of "telemetry" inside each plugin.

https://github.com/elastic/kibana/blob/d7cac22ba2dcfb8b1c1aaefad9cd7cac8aefca24/src/plugins/telemetry/README.md